### DISCUSSION TITLE
[Kavita] When clicking metadata within series page don't default to current library

### DISCUSSION THREAD

**Discussion by Cducharme84** [Kareadita/Kavita#2951] [Ideas]
### Idea Description

Currently when on a series page if you click any of the clickable metadata fields (think writer, artist etc) it leads to an instant creation of a filter looking for that metadata. However if only shows series by default from the current library.

So if the user's intent is to see all they have by a particular person, or by a publisher, or within a genre they would need to alter the filter by removing the entry that limits scope to one library. This is created as the bottom entry so to effectively do this we need to remove the filter we intended (genre etc) as it is the only one with a minus button, and then change the library limiting filter to our intended filter that was expected clicking the metadata name so we can see all examples within all libraries.

I do think this is backwards from a UX perspective, we should see the full scope first and then if we choose limit further with additional filters by library or whatever criteria is desired.

In the shots [here](https://imgur.com/a/AIZ0LW9) I clicked Frank Miller and would reasonably expect a collection wide search which could then be limited/filtered as I need instead I need to remove the top filter (the one I actually want) and change the bottom to that to achieve this.

Possible Solutions: 1. Remove the library limiting default.
                                   2. switch the fliter positioning so the library limiting filter can be removed without juggling.

### Idea Category

User Experience

### Duration of Using Kavita

1.25ish years

### Before submitting

- [X] I've already searched for existing ideas before posting.

**Comment by Cducharme84** [comment]
I placed here in ideas as I'm not sure if this was intended functionality and since it would be a fairly big behavioral change thought this the best venue for other user input.

**Comment by maximo1984** [comment]
I have brought up this issue prior. Utilizing this metadata is pointless currently, because the filter only leads you to a very large database of series. What if that person just happened to do a cover for a 300 issue series? Well, you are getting that full 300 series in your search. Pointless in current form.

**Reply by DieselTech** [reply]
Just because something doesn't work for your specific use case doesn't make it pointless. When this smart tag system was designed it was done as a first pass to get core functionality in so that users could use it and give feedback. There was next to no user input on these features except from a very small (3-5 people) group who helped work out the idea. Not enough users give insightful feedback on how they use the features to make them "perfect" on release.

**Reply by Cducharme84** [reply]
I'm not asking for issue based granular filtering (though that would be great if tags like characters or locations did work that way as they can be set in the ComicInfo.xml per issue) though, I'm asking overall to not default to solely the library you were in when you clicked the metadata or to have that part of the filter created by the mouse click be flipped so that the library is the top option since that's the one with a delete button which makes sense to not default to merging library types for a built in filter.

I do think there's merit to a separate idea for the issue based filtering, with some possible examples and possible ideas how it would look so the average user knows what's being filtered and where, would be a great way to show Joe and the team what you're looking for productively. I promise a comment on another idea isn't gonna get it on the consideration table though.

**Comment by majora2007** [comment]
This has been delivered as of the current nightly and the upcoming v0.8.3 stable. If this isn't meeting your expectation, please comment and I will reopen the feature request.

**Comment by Cducharme84** [comment]
It’s working great!
